### PR TITLE
user/git-extras: update to 7.2.0

### PR DESCRIPTION
**Pull request by ttyyls**

### CODE REVIEW

File: user/git-extras/template.py [hunk 1]
```diff
@@ -1,13 +1,47 @@
...
+            "bin/git-bulk",
+            "bin/git-extras",
+            "bin/git-force-clone",
+            "bin/git-fork",
+            "bin/git-get",
+            "bin/git-ignore",
+            "bin/git-setup",
+            "bin/git-standup",
+        ]:
+            pass
```
Comment (ttyyls) [inline user/git-extras/template.py:41]: this whole thing acts a subtitute to https://github.com/tj/git-extras/blob/main/Makefile#L68

Comment (nekopsykose) [reply user/git-extras/template.py:41]: why not just the makefile install for the entire thing again

Comment (ttyyls) [reply user/git-extras/template.py:41]: needs a patch.. which i'll add instead of trying to recreate it in the template.

**Comment by nekopsykose** [comment]
this installs to /etc/bash_completion which is wrong

**Comment by ttyyls** [comment]
> this installs to /etc/bash_completion which is wrong
patched it out and added the bash_completion install to post_install instead

**Comment by nekopsykose** [comment]
also the commit message should say `user/git-extras`

**Comment by ttyyls** [comment]
> pretty sure you don't need a patch and manual install, setting `COMPL_DIR` in `make_install_args` should be enough

im already patching the makefile anyway, I just thought i%'d be more consistent this way

**Comment by q66** [comment]
i think just adding git to hostdeps should avoid having to patch anything?

**Pull request merged**